admin/data/sql/migrations/20200210153747.undo.mod-check-table-type-and-spcreatecheck.sql

```diff
@@ -0,0 +1,139 @@
+-- DROP PROCEDURE IN ORDER TO RECREATE THE CHECK TABLE TYPE
+DROP PROCEDURE IF EXISTS [mtc_admin].[spCreateChecks];
+DROP TYPE IF EXISTS [mtc_admin].[checkTableType];
```

Comment:
Don't you need to recreate the TableType before defining the Procedure?

Reply:
@jon-shipley migrations have been updated please re-review.